inference-engine/tests/functional/shared_test_classes/src/single_layer/detection_output.cpp

```diff
@@ -60,76 +61,99 @@ std::string DetectionOutputLayerTest::getTestCaseName(const testing::TestParamIn
     result << "inH=" << attrs.input_height << "_";
     result << "inW=" << attrs.input_width << "_";
     result << "OS=" << attrs.objectness_score << "_";
+    result << "RDS=" << replaceDynamicShapesToIntervals << "_";
```

Comment:
Is not it better to print true or false instead of 0 and 1?

Reply:
done, thanks